region/brinstar/green/Green Hill Zone.json

```diff
@@ -336,7 +381,8 @@
           "HiJump",
           "h_canFly",
           "canWalljump",
-          "canUseFrozenEnemies"
+          "canUseFrozenEnemies",
+          "canTrickySpringBallJump"
```

Comment:
The "canTrickySpringBallJump" is already covered in the "Base" strat. A "canTrickySpringBallJump" would imply having Morph, so no need to turnaround aim cancel.